docs/man/man3/nxt_unit_response_add_content.3

```diff
@@ -0,0 +1,91 @@
+.\" (C) 2023, NGINX, Inc.
+.\"
+.TH nxt_unit_response_add_content 3 (date) "NGINX Unit (unreleased)"
+.SH Name
+nxt_unit_response_add_content \- add content to HTTP response in Unit app
+.SH Library
+NGINX Unit library
+.RI ( libunit ", " -lunit )
+.SH Synopsis
+.nf
+.B #include <nxt_unit.h>
+.PP
+.BI "int nxt_unit_response_add_content(nxt_unit_request_info_t *" req ,
+.BI "                                  const void " content [. size ],
+.BI "                                  uint32_t " size );
+.fi
+.SH Arguments
+.TP
+.I req
+Request object.
+.TP
+.I content
+Content of the response.
+It's a buffer of
+.I size
+bytes.
+.SH Description
+.MR nxt_unit_response_add_content 3
+adds content to an HTTP response.
+.PP
+The response should have been initialized previously with
+.MR nxt_unit_response_init 3 .
+.PP
+The response should have enough remaining buffer size available.
+If there's not enough,
+more buffer can be allocated with
+.MR nxt_unit_response_buff_alloc 3 .
```

Comment:
Typo in `buff`. There are a few more cases below.

Reply:
Thanks!